projects/packages/my-jetpack/_inc/components/action-button/index.tsx

```diff
@@ -230,7 +230,7 @@ const ActionButton: FC< ActionButtonProps > = ( {
 				};
 			case PRODUCT_STATUSES.USER_CONNECTION_ERROR:
 				return {
-					href: '#/connection?skip_pricing=true',
+					href: '&step=connect-user',
```

Comment:
It looks like you are changing the hash routes to query parameters. I'm not sure if this aligns with the project so can you share why you did it?

Reply:
Explained in https://github.com/Automattic/jetpack/pull/44469#issuecomment-3130476347